wg-docs-tools/meeting-notes/2019-04-19.md

```diff
@@ -0,0 +1,39 @@
+# Docs & Tools WG meeting 2019-04-19
+
+## Attendees
+
+* @felix
```

Comment:
```suggestion
* @felixrieseberg
```

Reply:
Hm, are we doing GitHub handles or Slack handles? I seem to have mixed the two up.

Reply:
I should probably just have the same across both, but as long as I’m the only Felix in the maintainers org I’m fine with either :P

Reply:
I just made it Slack handles this time.